content/docs/serverless/serverless-driver.md

```diff
@@ -6,10 +6,14 @@ subtitle: Learn how to connect to Neon from serverless and edge environments ove
 
 The [Neon serverless driver](https://github.com/neondatabase/serverless) is a low-latency Postgres driver for JavaScript and TypeScript that allows you to query data from serverless and edge environments over HTTP or WebSockets in place of TCP.
 
-The driver is a drop-in replacement for [node-postgres](https://node-postgres.com/), the popular npm `pg` package you may already be familiar with.
+You can use the driver as a drop-in replacement for [node-postgres](https://node-postgres.com/), the popular npm `pg` package you may already be familiar with. See [Using node-postgres Pool or Client](#using-node-postgres-pool-or-client).
 
 The driver's low-latency capability is due to [message pipelining and other optimizations](https://neon.tech/blog/quicker-serverless-postgres).
 
+<Admonition type="note">
+The maximum response size for queries is 10 MB as of version 0.6.0, increased from the 1 MB maximum supported in earlier versions of the driver.
+</Admonition>
```

Comment:
Sorry to be commenting again on this! This limitation applies only to http fetch queries using the `neon()` function (unless `poolQueryViaFetch` has been set to `true`, in which case it applies to `Pool.query` queries too).

Reply:
Sorry, I misunderstood. I moved this back to the neon() function section. I have left out the part about "poolQueryViaFetch". I understand that this is an Advanced configuration option that users usually won't need to touch: https://github.com/neondatabase/serverless/blob/main/CONFIG.md#advanced-configuration